crates/iota-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -572,8 +572,10 @@ pub struct AuthorityEpochTables {
     /// Transactions that are being deferred until some future time
     deferred_transactions: DBMap<DeferralKey, Vec<VerifiedSequencedConsensusTransaction>>,
 
-    /// Tables for recording state for RandomnessManager.
-
+    #[rustfmt::skip]
+    #[allow(clippy::empty_line_after_outer_attr)]
+    // Tables for recording state for RandomnessManager.
```

Comment:
Could maybe just do this?

```suggestion
    // Tables for recording state for RandomnessManager.
    //
```

Reply:
Or possibly two consecutive newlines

Reply:
PR is basically draft because I don't know what to do about this :(